src/gdal/ogr_file_format.cpp

```diff
@@ -69,23 +70,22 @@
 
 // IWYU pragma: no_forward_declare QFile
 
-
 namespace OpenOrienteering {
 
 namespace ogr {
-	
-	class OGRDataSourceHDeleter
```

Comment:
I wondered how you could delete this class without breaking existing code. The answer is that there is a second definition in ogr_file_format_p.h. Both definitions were identical, fortunately.
The lesson from this observation is that we should keep all these utilities together, in ogr_file_format_p.h.

Reply:
Need to correct myself: There was no duplication. You moved it to `ogr_file_format_p.h`. So everything is okay here.